Dockerfile

```diff
@@ -0,0 +1,31 @@
+# extend node alpine base image
+FROM node:8.4.0-alpine
+
+ENV RENOVATE_VERSION 9.45.10
```

Comment:
This is something I'd really like to see a tool like `renovate` handle as well.

I have `ENV HUGO_VERSION 0.26` hard-coded in one of my Docker projects. 😢

Reply:
Yes, this really is a pain and I would love to see an automated way to do that..